src/Visits.php

```diff
@@ -109,13 +109,24 @@ public function record()
 		{
 			return;
 		}
-
+		
 		$visits = new VisitModel();
 		$visit  = new Visit();
-
+		
 		// start the object with parsed URL components (https://secure.php.net/manual/en/function.parse-url.php)
 		$visit->fill(parse_url( current_url() ));
 
+		// Check if URI has been whitelisted from Visit check
+		if ($exclude_uris = $this->config->excludeUris)
+		{ 
+			foreach ($exclude_uris as $excluded)
+			{ 
+				if (preg_match('#^'.$excluded.'$#iu', $visit->path))
```

Comment:
Better to use CodeIgniter's helper function:
```suggestion
				if (url_is($excluded))
```

https://codeigniter4.github.io/CodeIgniter4/helpers/url_helper.html#url_is